core/domain/base_model_validators.py

```diff
@@ -69,7 +71,9 @@
 
 
 class ExternalModelFetcherDetails(python_utils.OBJECT):
-    """Value object providing the class and ids to fetch an external model."""
+    """Value object providing the class and ids to fetch an external model.
+    Note: For the UserSettingsModel use the UserSettingsModelFetcherDetails.
```

Comment:
Add a newline above this, and change it to:

```
NOTE TO DEVELOPERS: For UserSettingsModel, use the UserSettingsModelFetcherDetails class instead of this one.
```

Reply:
Done.